x/epochs/types/genesis.go

```diff
@@ -42,6 +43,15 @@ func DefaultGenesis() *GenesisState {
 			CurrentEpochStartTime:   time.Time{},
 			EpochCountingStarted:    false,
 		},
+		{
+			Identifier:              MINT_EPOCH,
+			StartTime:               time.Time{},
+			Duration:                time.Minute * 60,
```

Comment:
Should we just call this HOUR_EPOCH?

Reply:
we could, but I slightly prefer to name it in a way that scopes it clearly to a single use, to avoid the risk of anyone using it for other purposes